tests/src/test/scala/whisk/core/controller/test/RulesApiTests.scala

```diff
@@ -898,7 +898,7 @@ class RulesApiTests extends ControllerTestCommon with WhiskRulesApi {
         val aentity = BadEntity(namespace, aname())
         val rule = WhiskRule(namespace, aname(), afullname(namespace, aname().name), afullname(namespace, aname().name))
         val trigger = WhiskTrigger(namespace, rule.trigger.name)
-        val action = WhiskAction(namespace, rule.action.name, js("??"))
+        val action = WhiskAction(namespace, rule.action.name, js6("??"))
```

Comment:
Added a new jsDefault that maps to js6.  Will make it easy to update in the future as Markus suggested.

As for the empty exec that would not work everywhere I had called js6, but only in cases where it passes "" or "??".